.k8s/dev/service.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   ports:
     - port: 80
-      name: http
+      name: metrics
```

Comment:
This seems wrong and likely to break stuff. This is the app/web-app service. Should there not be a separate service for metrics?